rubin_sim/moving_objects/make_lsst_obs.py

```diff
@@ -82,7 +89,7 @@ def setup_args(parser=None):
         type=float,
         default=1.75,
         help="If using a circular footprint, this is the radius of the FOV (in degrees). "
-        "Default 1.75 deg.",
+        "Default 2.08 deg.",
```

Comment:
The default value for r_fov should be 1.75 degrees (sorry if this is the result of miscommunication on slack). 
The use of r_fov should be for defining the circular footprint, not for defining the matching tolerance for rough ephemerides. Can you please update the docstring to match the default value?

Reply:
Is there an "outfile name" at some point in the arguments here? Probably needs an update to match the .npz filetype now?